src/common/i18n/locales/en-US.json

```diff
@@ -1794,7 +1794,59 @@
       "what-delegated-HP-header": "What is Delegated Hive Power?",
-      "what-delegated-HP-body": "<p>When you delegate Hive power you make a portion of your HIVE Power available to another user. Delegated Hive Power increases Voting Powe, Curation Rewards and Recourse Credits of that account. When a delegation is cancelled the HIVE Power will only be available again after 7 days.</p>"
+      "what-delegated-HP-body": "<p>When you delegate Hive power you make a portion of your HIVE Power available to another user. Delegated Hive Power increases Voting Powe, Curation Rewards and Recourse Credits of that account. When a delegation is cancelled the HIVE Power will only be available again after 7 days.</p>",
+      "what-friends-feed-header": "What does the friends feed show?",
+      "what-friends-feed-body": "<p>Friends feed contains information about posts and reblogs of the accounts you follow.</p>",
+      "how-to-reblog-header": "How to Reblog post on Ecency?",
+      "how-to-reblog-body": "<p>Click on the Rectangular Arrow Icon under the post you like. Then choose Reblog and this post will appear in your Blog.</p>",
+      "what-is-reblog-header": "What is Reblog?",
+      "what-is-reblog-body": "<p>A reblog is when you see a post you enjoy on Ecency, and by clicking the reblog button, you make the post appear on your blog too. Now all of your followers can enjoy the post.</p>",
+      "how-sort-feed-header": "How-to-sort-Friends-feed?",
+      "how-sort-feed-body": "<p>Your Friends Feed contains posts published by your friends and their reblogs. If you only want to see the posts of your friends, you can select Exclude Reblog through the sorting function.</p>",
+      "what-trending-feed-header": "What does the Trending feed show?",
+      "what-trending-feed-body": "<p>Trending feed shows widely mentioned or discussed posts of entire platform.</p>",
+      "how-sort-trending-header": "How to sort Trending feed?",
+      "how-sort-trending-body": "<p>Trending feed can be sorted depending on what information you want to see. For example, you can choose one depending on whether you want to see only posts from My Community or Global posts from the entire platform.</p>",
```

Comment:
@Xurwi We can also add, if you wan to surf Trending feed on specific tag or topic, you can do so here as well by selecting topics from left side.